ci_scripts/run_analyzer.sh

```diff
@@ -33,7 +33,11 @@ fi
 
 # Search for warnings in log file
 echo "Searching for static analyzer warnings..."
-grep "warning:" "${log_file}" > "/dev/null"
+
+# Fun note:
+# xcodebuild outputs a line like "...SomeFile.m:36:1: warning: foo"
+# ...but sometimes, it inserts spurious newlines everywhere
```

Comment:
yuck.

Did you consider capturing the `xcpretty` output instead of `xcodebuild`? Seems like `The following commands produced analyzer issues:` could be a good alternative grep string, and might avoid the newline problem?

Reply:
Yeah, I did see xcpretty still outputs that.  Perhaps we should log that or both, if the raw logs are too.  

I defaulted to optimizing for the smallest change here 🤷‍♂